src/test/frames.test.ts

```diff
@@ -0,0 +1,30 @@
+import createFrames from '../logic/graphFrames';
+import { createFramesWithEdges } from '../logic/graphFrames';
+import { Algorithm, DisassembledGraph, Edge } from '../logic/commonInterfaces';
+
+const graph: DisassembledGraph = [[0, 1], [new Edge(0, 1)]];
+const alg: Algorithm = {
+  traversalHistory: [{ node: 0, lookups: [1] }],
+  paths: {
+    0: [],
+    1: [0],
+  },
+};
+
+const frames = createFrames(graph, alg)
```

Comment:
missing semicolon (do u have eslint on the job btw?)

Reply:
I have eslint but it seems it does not see tests.